scm/etc/scripts/UFS_forcing_ensemble_generator.py

```diff
@@ -0,0 +1,193 @@
+#!/usr/bin/env python
+
+###############################################################################
+# Dependencies
+###############################################################################
+import argparse
+import os
+from netCDF4 import Dataset
+import xarray as xr
+import numpy as np
+import random
+
+###############################################################################
+# Argument list
+###############################################################################
+parser = argparse.ArgumentParser()
+parser.add_argument('-d',    '--dir',           help='path to UFS Regression Test output', required=True)
+parser.add_argument('-n',    '--case_name',     help='name of case',                       required=True)
+parser.add_argument('-lonl', '--lon_limits',    help='longitude range, separated by a space', nargs=2,   type=float, required=False)
+parser.add_argument('-latl', '--lat_limits',    help='latitude range, separated by a space',  nargs=2,   type=float, required=False)
+parser.add_argument('-lons', '--lon_list',      help='longitudes, separated by a space',      nargs='*', type=float, required=False)
+parser.add_argument('-lats', '--lat_list',      help='latitudes, separated by a space',       nargs='*', type=float, required=False)
+parser.add_argument('-nens', '--nensmembers',   help='number of SCM UFS ensemble memebers to create',                 type=int,   required=False)
+parser.add_argument('-dt',   '--timestep',      help='SCM timestep, in seconds',                                      type=int,   default = 3600)
+parser.add_argument('-cres', '--C_RES',         help='UFS spatial resolution',                                        type=int,   default = 96)
+parser.add_argument('-sdf',  '--suite',         help='CCPP suite definition file to use for ensemble',                            default = 'SCM_GFS_v16')
+parser.add_argument('-sc',   '--save_comp',     help='flag to save a file with UFS data for comparisons',                         action='store_true')
+parser.add_argument('-near', '--use_nearest',   help='flag to indicate using the nearest UFS history file gridpoint, no regridding',action='store_true')
+
+###############################################################################
+# Main program
+###############################################################################
+def main():
+    # Get command line arguments
+    args  = parser.parse_args()
+
+    if (not args.dir):
```

Comment:
This shouldn't be necessary due to the `required=True` for the `dir` argument.

Reply:
Good point. I will remove.